azure-cli.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -e
+
+echo "Please insert the name of azure container registry :"
+read AZ_REGISTRY
+
+echo "Please insert the username Azure :"
+read AZ_USERNAME
+
+echo "Please insert the Tenant Azure :"
```

Comment:
```suggestion
echo "Azure tenant ID :"
```

Reply:
I'd move the "azure tenant" question before the username, to keep username/password next to each other.